### PR TITLE
Update with image references

### PR DESCRIPTION
**Pull request by kylelaffoon**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c5714fc24c/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a8abc1e87f/how-to/)

File: content/cloud-databases/Backup-Replication-Cloud-Databases.md [hunk 1]
```diff
@@ -25,25 +25,25 @@ Use the following steps to copy individual backups from one datacenter to anothe
 
 3.	In the **Backups** menu, select **MySQL Backups** under **MYSQL BACKUPS**.
 
-<! Insert Image: TopNavigationBackups.png>
+[](https://github.com/rackerlabs/rackspace-how-to/blob/master/_assets/img/cloud-databases/backup-replication-for-disaster-recovery/top_navigation_backups.png)
```
Comment (DonNateR) [inline content/cloud-databases/Backup-Replication-Cloud-Databases.md]: Image references should be similar to this, ``

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-96488f8816/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d0edebb9f4/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @kylelaffoon. The attached PDf contains my markup and comments. I added in the screenshots that I think we might keep. The first few don't seem necessary, but the last two, which show results specific to this new procedure, might be helpful. Anyway, please let me know if you have any questions. Thanks!

[Backup replication for disaster recovery with cloud databases_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/247417/Backup.replication.for.disaster.recovery.with.cloud.databases_editsKH.pdf)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-edd68dcbe6/how-to/).

**Comment by ReneeRendon** [comment]
looks good

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3bc70bab83/how-to/). :bow:

**Pull request merged**